File: Swift/프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.swift
import Foundation

func solution(_ numLog:[Int]) -> String {
    var answer = ""
    
    for i in 0..<(numLog.count - 1){
        let difference = numLog[i+1] - numLog[i]
        
        switch difference {
        case 1:
            answer += "w"
        case -1:
            answer += "s"
        case 10:
            answer += "d"
        case -10:
            answer += "a"
        default:
            break
        }
    }
    
    return answer
}

File: Python3/프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.py
def solution(numLog):
    answer = ''
    
    for i in range(len(numLog)-1):
        difference = numLog[i+1] - numLog[i]
        if difference == 1:
            answer += "w"
        elif difference == -1:
            answer += "s"
        elif difference == 10:
            answer += "d"
        else:
            answer+= "a"
            
    return answer